plugins/inputs/fireboard/fireboard.go

```diff
@@ -0,0 +1,126 @@
+package fireboard
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+// Fireboard gathers statistics from the fireboard.io servers
+type Fireboard struct {
+	AuthToken string
+	URL       string
+
+	client *http.Client
+}
+
+// NewFireboard return a new instance of Fireboard with a default http client
+func NewFireboard() *Fireboard {
+	tr := &http.Transport{ResponseHeaderTimeout: time.Duration(3 * time.Second)}
+	client := &http.Client{
+		Transport: tr,
+		Timeout:   time.Duration(4 * time.Second),
```

Comment:
It could be nice to expose the timeout as a configurable option.  However, this will make it difficult to initialize the http.Client here.  We recently added an optional Init() function that could be used for this and the config validation steps in Gather.
```
func (r *Fireboard) Init() error {
```